src/components/Onfido/index.js

```diff
@@ -0,0 +1,70 @@
+import './index.css';
+import React from 'react';
+import * as OnfidoSDK from 'onfido-sdk-ui';
+import onfidoPropTypes from './onfidoPropTypes';
+import CONST from '../../CONST';
+
+class Onfido extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onfidoOut = undefined;
+    }
+
+    componentDidMount() {
+        this.onfidoOut = OnfidoSDK.init({
+            token: this.props.sdkToken,
+            containerId: CONST.ONFIDO.CONTAINER_ID,
+            steps: [
+                {
+                    type: CONST.ONFIDO.TYPE.DOCUMENT,
+                    options: {
+                        useLiveDocumentCapture: true,
+                        forceCrossDevice: true,
+                        showCountrySelection: false,
+                        documentTypes: {
+                            national_identity_card: {
+                                country: null,
+                            },
+                            residence_permit: {
+                                country: null,
+                            },
+                            passport: true,
+                        },
+                    },
+                },
+                {
+                    type: CONST.ONFIDO.TYPE.FACE,
+                    options: {
+                        requestedVariant: CONST.ONFIDO.VARIANT.VIDEO,
+                        uploadFallback: false,
+                    },
+                },
+            ],
+            smsNumberCountryCode: CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE.US,
+            showCountrySelection: false,
+            onComplete: this.props.onSuccess,
+            onError: () => {
```

Comment:
Should we do some generic error handling here, maybe like this?

```suggestion
            onError: () => {
                this.props.onUserExit();
                Growl.show(SOME_GENERIC_ERROR_MESSAGE, CONST.GROWL.ERROR);
            }
```

Reply:
I didn't know we had grow, but sounds like a good use case for it!

Reply:
It's new 😁